### PR TITLE
article view

### PR DESCRIPTION
**Pull request by andyOlarte514**
## Summary
- View article

### Android
Screen Shot 2021-08-06 at 11 47 44 AM
Screen Shot 2021-08-06 at 11 48 21 AM

### iOS
Screen Shot 2021-08-06 at 11 53 51 AM
image

## Trello Card
- https://trello.com/c/60PaYCGH/13-view-article

### CODE REVIEW

File: src/app/screens/DetailArticle/index.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import { View, Image, TouchableOpacity, Text } from 'react-native';
+import CustomText from '@components/CustomText';
+import { Navigation } from '@interfaces/navigation';
+import { formatDate } from '@utils/dateUtils';
+import icAddInactive from '@assets/TabBar/icAddpostInactive.png';
+import icAddActive from '@assets/TabBar/icAddpostActive.png';
+import icFavouriteInactive from '@assets/TabBar/icFavoriteInactive.png';
+import icFavouriteActive from '@assets/TabBar/icFavoriteActive.png';
+
+import icDefaultArticleImage from '../Home/components/ArticleItem/assets/default_article_image.jpg';
```
Comment (john07289) [inline src/app/screens/DetailArticle/index.tsx:11]: Move default_article_image.jpg to assets and use '@assets/defaultArticleImage.jpg

File: src/app/screens/DetailArticle/index.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            {tagList.map((tag: string) => (
+              <Text style={styles.tag}>{tag}</Text>
+            ))}
+          </View>
+        )}
+        <View style={styles.separator} />
+        <CustomText>{title}</CustomText>
+        <CustomText label>{description}</CustomText>
+        <View style={styles.interactionButtons}>
+          <TouchableOpacity style={styles.interactionButton} onPress={() => setIsFollow(!isFollow)}>
```
Comment (john07289) [inline src/app/screens/DetailArticle/index.tsx:57]: I think that this contains testID

File: src/app/screens/DetailArticle/index.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            <Image
+              style={styles.interactionButtonImage}
+              source={isFollow ? icAddActive : icAddInactive}
+              resizeMode="contain"
+            />
+          </TouchableOpacity>
+          <TouchableOpacity style={styles.interactionButton} onPress={handleToggleFavorite}>
```
Comment (john07289) [inline src/app/screens/DetailArticle/index.tsx:64]: same

File: src/app/screens/Home/index.tsx [hunk 4]
```diff
@@ -21,7 +23,15 @@ function Home() {
...
   const renderSeparator = useCallback(() => <View style={styles.separator} />, []);
 
-  const renderItem: ListRenderItem<Article> = useCallback(({ item }) => <ArticleItem item={item} />, []);
+  const handleClickArticle = useCallback(
+    (article: Article) => navigation.navigate(Routes.DetailArticle, { article }),
+    [navigation]
+  );
+
+  const renderItem: ListRenderItem<Article> = useCallback(
+    ({ item }) => <ArticleItem item={item} onClick={handleClickArticle} />,
```
Comment (EstebanIndiveri) [inline src/app/screens/Home/index.tsx:32]: maybe you can change this name to onPress or something like this.

**Review by EstebanIndiveri — APPROVED** [review]
great job

**Review by john07289 — APPROVED** [review]

**Pull request merged**